stats/stats.go

```diff
@@ -261,3 +261,15 @@ func NewSubmetric(name string) (parentName string, sm Submetric) {
 	}
 	return parts[0], Submetric{Name: name, Tags: tags}
 }
+
+func (m *Metric) Summary() *Summary {
+	return &Summary{
+		Metric: m,
+		Samples: m.Sink.Format(),
```

Comment:
`Summary` maybe?

Also: for consistency, all field names should have camelCase names when serialized to/from JSON.